man/git-create-branch.md

```diff
@@ -7,17 +7,27 @@ git-create-branch(1) -- Create branches
 
 ## DESCRIPTION
 
-  Creates local branch named &lt;branchname&gt; and optionally sets up a remote tracking branch.
+Creates local branch named &lt;branchname&gt; and optionally sets up a remote tracking branch.
 
 ## OPTIONS
 
-  &lt;-r|--remote [remote_name]&gt;
+&lt;-r|--remote [remote_name]&gt;
 
-  Setup a remote tracking branch using `remote_name`. If `remote_name` is not supplied, use `origin` by default.
+Setup a remote tracking branch using `remote_name`. If `remote_name` is not supplied, use `origin` by default.
 
-  &lt;branchname&gt;
+&lt;branchname&gt;
 
-  The name of the branch to create.
+The name of the branch to create.
+
+## PREFERENCES
+
+You may save preferences for the `remote` option above by using `git config` with the key `git-extras.create-branch.remote` whose value works like the corresponding option. A value of `true` will use the default remote `origin`. Any other value will represent a remote name.
+
+    $ git config git-extras.create-branch.remote true
+
+    $ git config git-extras.create-branch.remote upstream
+
+With this preference set, `git-create-branch` will behave as if the `-r|--remote` option were given.
```

Comment:
We should mention that `-r|--remote` will override the preference.

Reply:
:+1: